test/integration/sagemaker/test_distributed_operations.py

```diff
@@ -35,6 +36,29 @@ def test_dist_operations_multi_gpu(sagemaker_session, ecr_image, dist_gpu_backen
     _test_dist_operations(sagemaker_session, ecr_image, instance_type, dist_gpu_backend, 1)
 
 
+@pytest.mark.skip_cpu
+def test_dist_operations_fastai_gpu(sagemaker_session, ecr_image, py_version):
+    if py_version != 'py3':
+        print('Skipping the test because fastai supports >= Python 3.6.')
+        return
```

Comment:
Why skip py3 in the test body but skip cpu in the pytest decorator?

Reply:
the only way i know how to skip based on a test fixture value is by creating another fixture automatically applied to all the tests which seems unnecessary in that case.

But I am open for a better work around suggestions.